spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateaddition">
+          <h1>CalendarDateAddition ( _calendar_, _date_, _duration_, _overflow_ )</h1>
+          <p>The abstract operation CalendarDateAddition performs implementation-defined processing to add _duration_ to _date_ in the context of the calendar represented by the string _calendar_ and returns the corresponding day, month and year of the result in the ISO calendar values as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedifference">
+          <h1>CalendarDateDifference ( _calendar_, _one_, _two_, _largestUnit_ )</h1>
+          <p>The abstract operation CalendarDateDifference performs implementation-defined processing to find the difference between the two dates _one_ and _two_ in the context of the calendar represented by the string _calendar_ and returns the corresponding years, months, weeks and days as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateera">
+          <h1>CalendarDateEra ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEra performs implementation-defined processing to find the era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateerayear">
+          <h1>CalendarDateEraYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEraYear performs implementation-defined processing to find the year of the current era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateyear">
+          <h1>CalendarDateYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateYear performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonth">
+          <h1>CalendarDateMonth ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonth performs implementation-defined processing to find the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthcode">
+          <h1>CalendarDateMonthCode ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonthCode performs implementation-defined processing to find the monthCode corresponding to the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
```

Comment:
Need to clarify the format of the month code: 
* First character is the literal character "M"
* Next 2 characters are the zero-padded month number of the current month in a normal (non-leap) year. For example, this number will be `05` for both Adar I and Adar II in a Hebrew leap year.
* If this is a leap month in a lunisolar calendar like Hebrew or Chinese, then a 4th character is added: the literal character "L". 

May want to add a few examples.

Reply:
As an exception to what I said above, I actually do agree that the description of the month code format belongs here.

Reply:
Should I describe the format in text or should I add it to the grammar and link to it here?

Reply:
Personally, I think it's OK to describe it in text.